pkg/cluster-operator/customcluster_controller_test.go

```diff
@@ -0,0 +1,100 @@
+package clusteroperator
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"kurator.dev/kurator/pkg/apis/infra/v1alpha1"
```

Comment:
the package 'kurator.dev/kurator/pkg/apis/infra/v1alpha1' needs to be separated from the others.
see https://github.com/kurator-dev/kurator/actions/runs/5797336703/job/15732862721#step:6:177